pyspark-notebook/Dockerfile

```diff
@@ -32,10 +32,23 @@ RUN apt-get update --yes && \
 
 # Spark installation
 WORKDIR /tmp
-RUN wget -q "https://archive.apache.org/dist/spark/spark-${APACHE_SPARK_VERSION}/spark-${APACHE_SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}-scala${scala_version}.tgz" && \
+
+RUN if [ -z "$scala_version" ] ; then \
+    RUN wget -q "https://archive.apache.org/dist/spark/spark-${APACHE_SPARK_VERSION}/spark-${APACHE_SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}.tgz" && \
```

Comment:
It may be cleaner to create `ARG SPARK_TGZ=...` in the if/else then use it below instead of duplicating the code.